airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -195,20 +198,20 @@ private static boolean isCdc(final JsonNode config) {
 
   @Override
   protected AirbyteStateType getSupportedStateType(final JsonNode config) {
-    if (!featureFlags.useStreamCapableState()) {
-      return AirbyteStateType.LEGACY;
-    }
+//    if (!featureFlags.useStreamCapableState()) {
```

Comment:
should these commented-out blocks just be deleted? (also, is it intended that we'll stop respecting the feature flag?)